django/db/backends/postgresql/schema.py

```diff
@@ -104,6 +104,12 @@ def _alter_column_type_sql(self, table, old_field, new_field, new_type):
 
     def _alter_field(self, model, old_field, new_field, old_type, new_type,
                      old_db_params, new_db_params, strict=False):
+        if old_field.db_index:
```

Comment:
Would it be better to combine the if statements with `and`? Could you add a comment above the `if` similar to the other comments in this method?